virttest/utils_test/libvirt.py

```diff
@@ -3665,6 +3665,33 @@ def check_qemu_cmd_line(content, err_ignore=False):
     return True
 
 
+def run_remote_cmd(cmd, params, remote_runner=None, ignore_status=True):
```

Comment:
It is better to place this method in `remote.py`

Reply:
updated